packages/client/src/components/DebitDetailsCard/DebitDetailsCard.tsx

```diff
@@ -0,0 +1,33 @@
+/**
+ * @author David Zhu <dzhu292@aucklanduni.ac.nz>
+ */
+
+import Card from '../Card/Card';
+import DebitDetailsCardProps from './DebitDetailsCardProps';
+import { MdContentCopy } from 'react-icons/md';
+
+
+const DebitDetailsCard = (props: DebitDetailsCardProps) => {
+    const body = props.text.map((text, index) => 
+        <p className="font-normal text-gray-500" key={index}>{text}</p>); // TODO: redundant
+    if (typeof(props.sessionId) != "undefined") {
+        body.push(<br/>,
+        <p className="font-normal text-gray-500 inline">SessionID: </p>, // TODO: try not to use this
+        <p className="font-bold text-gray-500 inline">{props.sessionId}</p>)
+    }
+    if (typeof(props.copy) != "undefined") { // dont need bg-change for mobile devices but can be used for desktop
```

Comment:
This can actually be simplified down to if (props.sessionId) ... because if it is null or undefined it will be 'false'